### PR TITLE
Add get account detail query

### PR DESCRIPTION
**Pull request by MizukiSonoko**
## What is this pull request?
Add new function of query
- Add get account detail query
   
## Why do you implement it? Why do we need this pull request?
Get account information from ledger

## Details/Features
List of features / major commits
- [x] Add GetAccountDetail to proto/model
- [x] Add converter, executor to model
- [x] Add documents

### CODE REVIEW

File: irohad/model/converters/pb_query_response_factory.hpp [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #include <responses.pb.h>
 #include <model/account_asset.hpp>
 #include <model/queries/responses/account_assets_response.hpp>
+#include <model/queries/responses/account_detail_response.hpp>
 #include <model/queries/responses/account_response.hpp>
```
Comment (l4l) [inline irohad/model/converters/pb_query_response_factory.hpp]: Wrong brackets :( should be `"..."`

File: irohad/model/impl/query_execution.cpp [hunk 2]
```diff
@@ -98,6 +99,14 @@ bool iroha::model::QueryProcessingFactory::validate(
                          can_get_my_acc_ast, can_get_all_acc_ast);
 }
 
+bool iroha::model::QueryProcessingFactory::validate(
+  const model::GetAccountDetail& query) {
+  // TODO: check signatures
```
Comment (l4l) [inline irohad/model/impl/query_execution.cpp]: Jira issue?

File: irohad/model/queries/get_account_detail.hpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef IROHA_GET_ACCOUNT_DETAIL_HPP
+#define IROHA_GET_ACCOUNT_DETAIL_HPP
+
+#include <model/query.hpp>
```
Comment (l4l) [inline irohad/model/queries/get_account_detail.hpp]: Brackets again

File: irohad/model/converters/pb_query_response_factory.hpp (hunk 1 above)
Comment (MizukiSonoko) [reply irohad/model/converters/pb_query_response_factory.hpp]: How about other? (Imo, this pr's task is Add get account detail query. so I don't want to update other code)

Comment (l4l) [reply irohad/model/converters/pb_query_response_factory.hpp]: not really huge changes so it's fine

File: irohad/model/permissions.hpp [hunk 4]
```diff
@@ -54,8 +54,9 @@ namespace iroha {
     const std::string can_get_domain_signatories = "CanGetDomainSignatories";
 
     const std::string can_get_my_acc_ast = "CanGetMyAccountAssets";
-    const std::string can_get_all_acc_ast = "CanGetAllAccountAssets";
-    const std::string can_get_domain_acc_ast = "CanGetDomainAccountAssets";
+      const std::string can_get_all_acc_ast = "CanGetAllAccountAssets";
```
Comment (grimadas) [inline irohad/model/permissions.hpp]: You can also rebase onto develop, and add domain permissions

**Comment by l4l** [comment]
May you remove every merge commits (c7ec7bb2 and 9c43b5b) from this pr and rebase it?

**Comment by MizukiSonoko** [comment]
Sorry I miss again, I will re pr 🙇

**Pull request closed without merging**